sunpy_sphinx_theme/sunpy/static/style.css

```diff
@@ -205,15 +204,9 @@ p.whatsnew {
 
 /* Sidebar */
 
-#sidebar {
-  z-index: 100;
```

Comment:
This still makes me nervous. You *sure* this works?

Reply:
Not 100% sure.